bin/htmlproofer

```diff
@@ -71,6 +71,8 @@ Mercenary.program(:htmlproofer) do |p|
 
     options[:cache] = {}
     options[:cache][:timeframe] = opts['timeframe'] unless opts['timeframe'].nil?
+    
+    options[:http_status_ignore] = Array(options[:http_status_ignore]).map(&:to_i)
```

Comment:
@gjtorikian the input isn't `"421, 422"`, it's `["421", "422"]` I'll add a test.